packages/flutter/lib/src/rendering/layer.dart

```diff
@@ -1991,37 +2047,31 @@ class FollowerLayer extends ContainerLayer {
   Matrix4 _invertedTransform;
   bool _inverseDirty = true;
 
-  Offset _transformOffset<S>(Offset regionOffset) {
+  Offset _transformOffset<S>(Offset localPosition) {
     if (_inverseDirty) {
       _invertedTransform = Matrix4.tryInvert(getLastTransform());
       _inverseDirty = false;
     }
     if (_invertedTransform == null)
       return null;
-    final Vector4 vector = Vector4(regionOffset.dx, regionOffset.dy, 0.0, 1.0);
+    final Vector4 vector = Vector4(localPosition.dx, localPosition.dy, 0.0, 1.0);
     final Vector4 result = _invertedTransform.transform(vector);
     return Offset(result[0] - linkedOffset.dx, result[1] - linkedOffset.dy);
   }
 
   @override
-  S find<S>(Offset regionOffset) {
-    if (link.leader == null) {
-      return showWhenUnlinked ? super.find<S>(regionOffset - unlinkedOffset) : null;
-    }
-    final Offset transformedOffset = _transformOffset<S>(regionOffset);
-    return transformedOffset == null ? null : super.find<S>(transformedOffset);
-  }
-
-  @override
-  Iterable<S> findAll<S>(Offset regionOffset) {
+  @protected
+  bool findAnnotations<S>(AnnotationResult<S> result, Offset localPosition, { @required bool onlyFirst }) {
     if (link.leader == null) {
-      return showWhenUnlinked ? super.findAll<S>(regionOffset - unlinkedOffset) : <S>[];
+      if (showWhenUnlinked)
+        return super.findAnnotations(result, localPosition - unlinkedOffset, onlyFirst: onlyFirst);
```

Comment:
nit: You might add curly braces around this, to be consistent with the one below.